src/tests/ftest/pool/pool_svc.py

```diff
@@ -148,7 +149,9 @@ def test_pool_svc(self):
                         "Error stopping a pool non-leader - "
                         "DaosServerManager.stop_ranks([{}])".format(non_leader))
 
+                self.pool.wait_for_rebuild(False, interval=1)
                 # Verify the pool leader has not changed
                 self.check_leader(pool_leader, False)
+                self.destroy_pools(self.pool)
```

Comment:
Agree with Saurabh. Any reason why we destroy the pools here. Teardown should handle it.

Reply:
Just answered to Saurabh in this comment: https://github.com/daos-stack/daos/pull/5358#issuecomment-820772434 (it should be just above this one).